.storybook/storybookStyles.scss

```diff
@@ -17,6 +17,13 @@
   }
 }
 
+[data-theme='cmsgov'] {
+  .utility-example {
+    background-color: var(--color-secondary-lightest);
+    outline: 2px dashed var(--color-secondary);
+  }
+}
+
 [data-theme='medicare'] {
   @import '@cmsgov/ds-medicare-gov/src/styles/index';
   .utility-example {
```

Comment:
Are any of these rules needed anymore now that we use CSS variables? Looks like some of these may have been importing the variables and then using them in a rule for `.utility-example`, but now I'd think they just automatically adjust using the one rule at the top.